packages/zone.js/lib/common/events.ts

```diff
@@ -386,6 +404,30 @@ export function patchEventTarget(
     const unpatchedEvents: string[] = (Zone as any)[zoneSymbol('UNPATCHED_EVENTS')];
     const passiveEvents: string[] = _global[zoneSymbol('PASSIVE_EVENTS')];
 
+    function copyEventListenerOptions(options: any) {
+      if (typeof options === 'object') {
```

Comment:
Presubmit indicated that there are cases when `options` is `null`, which makes it through this check and fails at the `options.signal` read below. We should add an extra check, for example:
```suggestion
      if (typeof options === 'object' && options !== null) {
```

Reply:
Updated this check.